tests/test_mc_simulator.py

```diff
@@ -419,3 +445,154 @@ def test_MantarrayMCSimulator__handles_reads_of_size_greater_than_next_packet_in
 
     actual_2 = simulator.read(size=1)
     assert actual_2 == bytes(0)
+
+
+def test_MantarrayMCSimulator__raises_error_if_unrecognized_module_id_sent_from_pc(
+    mantarray_mc_simulator_no_beacon, mocker
+):
+    mocker.patch(
+        "builtins.print", autospec=True
+    )  # don't print all the error messages to console
+
+    _, _, _, _, simulator = mantarray_mc_simulator_no_beacon
+
+    dummy_timestamp = 0
+    dummy_packet_type = 1
+    test_module_id = 254
+    test_handshake = create_data_packet(
+        dummy_timestamp,
+        test_module_id,
+        dummy_packet_type,
+        DEFAULT_SIMULATOR_STATUS_CODE,
+    )
+
+    simulator.write(test_handshake)
+    time.sleep(QUEUE_CHECK_TIMEOUT_SECONDS)  # let input queue get populated
```

Comment:
did we want to have the simulator take a kwarg of how much time (default `None`) to sleep after a `.write`, and supply an argument during test cases when instantiating the simulator? or did we determine it was better to have it as `sleep` lines in the test cases...I don't recall the outcome of that discussion

Reply:
I'm not sure we ever had that discussion, but I think it could be nice to have the simulator sleep on it's own during test cases